crates/blockifier/src/scale_codecs.rs

```diff
@@ -29,18 +36,20 @@ pub struct HashSetCodec<T>(Vec<T>)
 where
     T: Encode + Decode;
 
-impl<T: Encode + Decode + Eq + Hash> From<HashSetCodec<T>> for HashSet<T> {
+impl<T: Encode + Decode + Eq + Hash> From<HashSetCodec<T>> for HashSet<T, HasherBuilder> {
```

Comment:
Je suis pas sur que ça soit la solution.
Je pense il vaut mieux remplace the HashSet par une IndexMap